lib/python/Components/Converter/Converter.py

```diff
@@ -5,9 +5,13 @@ class Converter(Element):
 	def __init__(self, arguments):
 		Element.__init__(self)
 		self.converter_arguments = arguments
+		self.separatorChar = ""
 
 	def __repr__(self):
 		return "%s(%s)" % (str(type(self)), self.converter_arguments)
 
 	def handleCommand(self, cmd):
 		self.source.handleCommand(cmd)
+
+	def appendToStringWithSeparator(self, str, part):
+		return part if str == "" else f"{str}{self.separatorChar}{part}"
```

Comment:
Here you have to add spaces around separator since skin engine strips them if you define them in convertor parameter

Reply:
There is new implementation of that. You can check it out.